flink-streaming-scala/src/test/scala/org/apache/flink/streaming/api/scala/DataStreamTest.scala

```diff
@@ -649,6 +651,18 @@ class DataStreamTest extends AbstractTestBase {
     assert(sg.getIterationSourceSinkPairs.size() == 2)
   }
 
+  @Test
+  def testReinterpretAsKeyedStream(): Unit = {
```

Comment:
So far, this file is a suite of tests, we never execute Flink programs, which should only happen in ITCases, IMO.